volto/bootstrap.rst

```diff
@@ -0,0 +1,89 @@
+.. _bootstrap-label:
+
+=============================
+Bootstrapping A Volto Project
+=============================
+
+Installing Plone
+================
+
+On order to run Volto you need a backend. This can be either Plone or
+Guillotina. For this course we will use Plone, you can download Plone at
+:file:`https://plone.org/download`.
+
+Installing dependencies
+=======================
+
+First step is to install the correct Node version using :file:`nvm`:
+
+.. code-block:: console
+
+    $ curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
+
+Then you can install the latest LTS version of node:
+
+.. code-block:: console
+
+    $ nvm install --lts
+
+We use the package manager :file:`yarn`, to install do:
+
+.. code-block:: console
+
+    $ npm install -g yarn
+
+And add the :file:`create-volto-app` package:
+
+.. code-block:: console
+
+    $ yarn global add @plone/create-volto-app
+
+Bootstrapping A Project
+=======================
+
+To create a new volto project type the following:
+
+.. code-block:: console
+
+    $ create-volto-app my-volto-app
+
+It will create a folder called `my-volto-app` inside the current folder with the following structure:
+
+::
```

Comment:
Use an explicit `.. code-block::` with `:language: text` option, because the lexer might barf with the default of Python.

Do you try to build the docs locally before pushing a PR? If not, see https://training.plone.org/5/mastering-plone/about_mastering.html#building-the-documentation-locally